src/components/form/form-date/form-date.tsx

```diff
@@ -29,16 +36,30 @@ const FormDate: React.FC<FormDateProps> = ({
 
   const handleChange = (date: Date | undefined) => setFieldValue(name, date);
 
+  const handleManualEdit = (
+    e: NativeSyntheticEvent<TextInputEndEditingEventData>
+  ) => {
+    const { text } = e.nativeEvent;
+    const date = dayjs(text);
+    if (date.isValid() && date.year() > 1800) {
```

Comment:
Please take 1800 to a constant and... Do you think we should allow immortals to use our app?